test/run_tests.py

```diff
@@ -452,11 +475,22 @@ def do_dirty_check(test_base_ref_dir):
         print("Failed to run 'git clean --dry-run -d -x -e __pycache__'")
     stdout = try_decode(result.stdout)
     if len(stdout) != 0:
-        print("Error: Test dir is not clean, this can result in false negatives. To ignore this and continue anyway pass --ignore_dirty")
+        print("Error: Test dir is not clean, this can result in false negatives. To ignore this and continue anyway pass --ignore_dirty or pass --clean_dirty to clean")
         print("'git clean --dry-run -d -x -e __pycache__' output:\n---")
         print(stdout)
         sys.exit(1)
 
+def clean_dirty(test_base_ref_dir):
+    result = subprocess.run(
+        "git clean -f -d -x -e __pycache__".split(),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        cwd=test_base_ref_dir,
+        timeout=10)
+    return_code = result.returncode
+    if return_code != 0:
+        print("Failed to run 'git clean -f -d -x -e __pycache__'")
```

Comment:
move this string to a var since its being used twice

Reply:
done